### PR TITLE
chore: the column with to create weekView

### PR DESCRIPTION
**Pull request by Julian-B90**
Set the columnWidth for the new feature from calendar-utils: https://github.com/mattlewis92/calendar-utils/pull/44

And fix issue #1447

### CODE REVIEW

File: projects/demos/app/demo-modules/day-view-scheduler/day-view-scheduler.component.ts [hunk 1]
```diff
@@ -83,7 +83,8 @@ export class DayViewSchedulerCalendarUtils extends CalendarUtils {
   templateUrl: 'day-view-scheduler.component.html',
   providers: [DayViewSchedulerCalendarUtils],
 })
-export class DayViewSchedulerComponent extends CalendarWeekViewComponent
+export class DayViewSchedulerComponent
```
Comment (billyjov) [inline projects/demos/app/demo-modules/day-view-scheduler/day-view-scheduler.component.ts:86]: Can we pls avoid style change here ?

**Comment by billyjov** [comment]
Thanks you for this PR. Just left a minor change request

File: projects/demos/app/demo-modules/day-view-scheduler/day-view-scheduler.component.ts (hunk 1 above)
Comment (Julian-B90) [reply projects/demos/app/demo-modules/day-view-scheduler/day-view-scheduler.component.ts:86]: Hey sure :-). I have push the fix for the code styling.

Comment (billyjov) [reply projects/demos/app/demo-modules/day-view-scheduler/day-view-scheduler.component.ts:86]: Style have the old styles.

**Comment by Julian-B90** [comment]
We need first the new version from calendar-utils before trafis can complet.

**Comment by mattlewis92** [comment]
https://github.com/mattlewis92/angular-calendar/issues/1447#issuecomment-852023027

**Pull request closed without merging**